docs/source/using-executorch-windows.md

```diff
@@ -0,0 +1,149 @@
+# How to Build ExecuTorch for Windows
+
+This document outlines the current known working build instructions for building and validating ExecuTorch on a Windows machine.
+
+This demo uses the
+[MobileNet v2](https://pytorch.org/vision/main/models/mobilenetv2.html) model to
```

Comment:
I'd maybe change the wording here slightly since the current runner doesn't have camera integration. Maybe replace with "uses the MobileNet v2 model to classify images.".